sdks/java/io/iceberg/src/test/java/org/apache/beam/sdk/io/iceberg/catalog/IcebergCatalogBaseIT.java

```diff
@@ -216,9 +222,9 @@ public void cleanUp() throws Exception {
 
   @Rule
   public transient Timeout globalTimeout =
-      Timeout.seconds(OPTIONS.getRunner().equals(DirectRunner.class) ? 180 : 20 * 60);
+      Timeout.seconds(OPTIONS.getRunner().equals(DirectRunner.class) ? 300 : 20 * 60);
 
-  private static final int NUM_SHARDS = 10;
+  private static final int NUM_SHARDS = 1;
```

Comment:
Will this remove test coverage of sharded case? We can decrease shard / number of records on direct runner but good to keep parallelism on other runners (Dataflow)

cc: @ahmedabu98

Reply:
Done